examples/src/App.js

```diff
@@ -128,7 +128,7 @@ class App extends React.Component {
     const className = konamiTime ? 'viewport konamiTime' : 'viewport';
 
     return (
-      <Fragment>
+      <React.StrictMode>
```

Comment:
🐥Switch to using `React.StrictMode` to simulate the conditions that trigger the deprecation warning when the legacy context API is used.